Dockerfile.ci

```diff
@@ -1,8 +1,7 @@
-FROM ocaml/opam-dev@sha256:1dd4440b3e5f182f705cb5a74f9d4e860c2842b45ed72c199de89a894d13f522
-#FROM ocaml/opam-dev:alpine-3.5_ocaml-4.04.0
+FROM ocaml/opam:alpine-3.5_ocaml-4.04.0
```

Comment:
Why the move back to opam 1?

Reply:
The CI changes are now worked on #612 where I reverted that change. The plan is to first merge #612 and then rebase this PR on master with just the "newline" commit on it.